test/living-dom/node-layouting.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
@domenic Ah, thanks and apologies. I've rewritten these as web platform tests and I'll push them up now - please let me know if I've missed anything.